### PR TITLE
Improve tx page

### PR DESCRIPTION
**Pull request by selankon**
It reimplements some components

- DetailsGrid c630875
- QueryParamsTabs 5f0ff6e
- It fixes the ReducedTextAndCopy to be only one responsive component ecba81b (before it was two, now i merged into one)
- Also implements link mode on ReducedTextAndCopy d3b3943 and 9408fd0
- In addition it refactors the process card to show the name of the organization and the organization id at the same line 629b228. 
- It does a minor fix for responsive layout on list pages 047a4cf

### CODE REVIEW

File: src/components/Blocks/BlockCard.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const BlockCard = ({ height, time, proposer }: { height: number; time: st
             <Box fontSize={'sm'}>
               <Flex gap={2} align={'center'}>
                 <Trans i18nKey='blocks.proposer'>Proposer:</Trans>
-                <ReducedTextAndCopy color={'textAccent1'} toCopy={proposer}>
+                <ReducedTextAndCopy reduced={true} color={'textAccent1'} toCopy={proposer}>
```
Comment (elboletaire) [inline src/components/Blocks/BlockCard.tsx]: no need to do `={true}`. A prop without arguments is considered to be true.

```suggestion
                <ReducedTextAndCopy reduced color={'textAccent1'} toCopy={proposer}>
```

File: src/components/Organizations/Card.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const OrganizationCardSkeleton = ({ electionCount: ec, ...rest }: IOrganizationC
           ) : (
             <OrganizationName fontWeight={'bold'} wordBreak='break-all' size='sm' />
           )}
-          <ReducedTextAndCopy color={'textAccent1'} toCopy={pid}>
+          <ReducedTextAndCopy reduced={true} color={'textAccent1'} toCopy={pid}>
```
Comment (elboletaire) [inline src/components/Organizations/Card.tsx]: I counted four more occurrences (no need to review each one IMO)

File: src/components/Blocks/Detail.tsx [hunk 3]
```diff
@@ -1,40 +1,15 @@
...
-      <GridItem>
-        <Text fontWeight={'bold'}>{label}</Text>
-      </GridItem>
-      <GridItem>{children}</GridItem>
-    </>
-  )
-}
+import { DetailsGrid, GridItemProps } from '~components/Layout/DetailsGrid'
+import { QueryParamsTabs } from '~components/Layout/QueryParamsTabs'
+import { ReducedTextAndCopy } from '~components/Layout/CopyButton'
```
Comment (elboletaire) [inline src/components/Blocks/Detail.tsx]: Doesn't look properly sorted

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: src/components/Blocks/BlockCard.tsx (hunk 1 above)
Comment (selankon) [reply src/components/Blocks/BlockCard.tsx]: Reduced is not used anymore 62cb7e5

File: src/components/Organizations/Card.tsx (hunk 2 above)
Comment (selankon) [reply src/components/Organizations/Card.tsx]: Reduced is not used anymore 62cb7e5

File: src/components/Blocks/Detail.tsx (hunk 3 above)
Comment (selankon) [reply src/components/Blocks/Detail.tsx]: e145d7f

**Review by elboletaire — APPROVED** [review]

**Pull request merged**